CHANGELOG.md

```diff
@@ -1,66 +1,118 @@
 # Lotus changelog
 
-# 1.19.0-rc2 / 2022-11-23
+# 1.19.0 / 2022-12-07
 
-This is the 2nd release candidate for the upcoming OPTIONAL v1.19.0 release of Lotus. 
-More detailed changelog and description will come later
+This is an optional feature release of Lotus. This feature release includes the production ready SplitStore feature, the experimental Lotus node cluster feature, as well as numerous enhancments and bugfixes.
 
-## Changes
+## Highlights
+### 🟢 SplitStore v2（Production Ready!!) 🟢
 
-- feat: wdpost: Configurable pre-check timeouts
-- fix/build: Update Zondax/hid to 0.9.1
-- chore: update lotus version to 1.19.0-rc1
-- chore: merge release into master ([filecoin-project/lotus#9657](https://github.com/filecoin-project/lotus/pull/9657))
-- Raft consensus for lotus nodes in a cluster ([filecoin-project/lotus#9294](https://github.com/filecoin-project/lotus/pull/9294))
+Splitstore aims to reduce the node performance impact that's caused by the Filecoin's very large, and continuously growing chain datastore by having a hot and cold blockstore. You can find more about the Splitstore implementation [here](https://github.com/filecoin-project/lotus/blob/master/blockstore/splitstore/README.md).
+Splitstore has three basic modes for node operators to configure according to your needs: 
+- `discard`: hotstore only, automatically archive out-of-scope objects that are beyond 4 finalities(3600 epochs).
+- `universal`:  stores all chain data that's beyond 4 finalities into coldstore.
+- `messages`: only stores on-chain messages into coldstore.
+
+In this release, splitstore is enabled by default with `discard` mode. The `EnableColdStoreAutoPrune=` configuration is being deprecated in this release, as there is only ever one compaction running.
+There are more configuration one may set, you can read the full documentation about the SplitStoreV2 here: https://lotus.filecoin.io/lotus/configure/splitstore/.
```

Comment:
How do we make sure we don't break existing archival nodes?

Enabling splitstore on large existing nodes will likely result in never-ending compaction. It will also mess with full-nodes. Am I missing some smarter migration we're doing?